fix(native): title casing in filtering fast-fail alert box (#5526)
getsentry/sentry-docs: src/platform-includes/configuration/before-send/native.mdx

</Alert>

> <Alert level="warning" title="Bypassed in Crashpad on Windows When Capturing Fast-fail Crashes">

Reviewer: Here's a slightly different approach (and I've also tried to shorten it a bit)
```suggestion
<Alert level="warning" title="Limitations in Crashpad on Windows for Fast-fail Crashes">
```

> The Crashpad backend on Windows supports fast-fail crashes, which bypass SEH (Structured Exception Handling) primarily for security reasons. `sentry-native` registers a WER (Windows Error Reporting) module, which signals the `crashpad_handler` to send a minidump when a fast-fail crash occurs. But since this process bypasses SEH, the application local exception handler is no longer invoked, which also means that for these kinds of crashes, `before_send` and `on_crash` will not be invoked before sending the minidump and thus have no effect.

Reviewer: ```suggestion
The Crashpad backend on Windows supports fast-fail crashes, which bypass SEH (Structured Exception Handling) primarily for security reasons. `sentry-native` registers a WER (Windows Error Reporting) module, which signals the `crashpad_handler` to send a minidump when a fast-fail crash occurs. However, since this process bypasses SEH, the application local exception handler is no longer invoked, which also means that for these kinds of crashes, `before_send` and `on_crash` will not be invoked before sending the minidump and thus have no effect.
```
